### PR TITLE
Add osu defines when needed

### PR DESCRIPTION
**Pull request by tjoly-ledger**
## Description

Add OSU definitions when needed

## Changes include

- [X ] New feature (non-breaking change that adds functionality)

### CODE REVIEW

**Comment by xchapron-ledger** [comment]
IMO this solution seem fragile, when updating SDK files, we are going to often break the OSU build as OSU build is not check by the SDK CI...

**Comment by tjoly-ledger** [comment]
> IMO this solution seem fragile, when updating SDK files, we are going to often break the OSU build as OSU build is not check by the SDK CI...

Use existing flag BOLOS_OS_UPGRADER_APP instead of OSU to not introduce an other FLAG.

**Review by jarevalo-ledger — APPROVED** [review]
LGTM

File: include/os_io.h [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #pragma once
+#include "app_config.h"
```
Comment (xchapron-ledger) [inline include/os_io.h:2]: Could be below toolchain includes and just above `#include "bolos_target.h"`?

Comment (tjoly-ledger) [reply include/os_io.h]: done

**Review by xchapron-ledger — APPROVED** [review]

**Pull request merged**